pkg/config/config.go

```diff
@@ -164,6 +164,8 @@ type V3ioConfig struct {
 	BuildInfo *BuildInfo `json:"buildInfo,omitempty"`
 	// Override nginx bug
 	DisableNginxMitigation bool `json:"disableNginxMitigation,omitempty"`
+	// explicitly always use client aggregation
+	UsePreciseAggregations bool `json:"usePreciseAggregations,omitempty"`
```

Comment:
So why not call it `UseClientAggregations`?

Reply:
thought this would be clearer for the user. The user dont care/realy know about server vs client aggregates but he does know what a Precise aggregation is. 
WDYT?

Reply:
I think we can use this name and Ask @Sharon-iguazio to document the implications